app/components/Form/ProjectEmissionIntensityReportFormSummary.tsx

```diff
@@ -177,7 +177,7 @@ const ProjectEmissionsIntensityReportFormSummary: React.FC<Props> = (props) => {
           summaryEmissionIntensityReport?.newFormData
             ?.measurementPeriodStartDate,
           summaryEmissionIntensityReport?.newFormData?.measurementPeriodEndDate,
-          false
+          true
```

Comment:
This causes `Production Functional Unit` not showing up on the summary page and one of your e2e tests to fail.

Reply:
Great catch. Thank you!